src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -122,24 +123,27 @@ const char* GetCalendarName(CalendarId calendarId)
 		case JAPANESELUNISOLAR:
 		case TAIWANLUNISOLAR:
 			return "chinese";
```

Comment:
> •ChineseCalendar is a concrete subclass of Calendar that implements a traditional Chinese calendar. The traditional Chinese calendar is a lunisolar calendar

we need to be careful here as it could be the name is deceiving. we want to make sure the Chinese calendar implementation in Linux is matching our Lunisolar calendar. my understanding is our ChineseLunisolar calendar is really supporting some special years cycles which is not really used in such regions. we need to fix out first how the Chinese calendar supported in such regions before we supply ours which can be different